public/locations-list-map.js

```diff
@@ -405,6 +656,13 @@ function centerMapToBounds(map, bounds, maxZoom) {
   }
 }
 
+// TODO (patricknelson: The code inside the .getCurrentPosition() is now DUPLICATED in centerMapToMarkersNearUser() AND centerMapToMarkersNearCoords()
```

Comment:
Happy to see a targeted follow-up PR for de-dups.

Reply:
Should I leave this comment here? Seems worthwhile warning folks that the code is duplicated. LMK your opinion, though.

Reply:
Actually, I'll just leave a note here that this is duplicated and then leave out the opinion on how to fix.

Reply:
Leaving in place as a mild warning